src/jit/utils.cpp

```diff
@@ -353,7 +353,7 @@ void                dspRegMask(regMaskTP regMask, size_t minSiz)
 #endif // _TARGET_*
             }
             // We've already printed a register. Is this the end of a range?
-            else if ((regNum == REG_INT_LAST)
+            else if (regNum == REG_INT_LAST
```

Comment:
The reason was the same as for the first case. I understand that it violates the coding conventions, so if you think that duplicating the term in #else branch is better I'll do it that way.